core/src/main/java/org/elasticsearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -937,6 +937,10 @@ protected int doHashCode() {
     protected Query doToQuery(QueryShardContext context) throws IOException {
         //TODO would be nice to have all the settings in one place: some change though at query execution time
         //e.g. field names get expanded to concrete names, defaults get resolved sometimes to settings values etc.
+        if (splitOnWhitespace == false && autoGeneratePhraseQueries) {
+            throw new IllegalArgumentException("it is disallowed to disable split_on_whitespace " +
+                "if auto_generate_phrase_queries is activated");
```

Comment:
can you put brackets around configuration options to be consistent with most of our error messages?